atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -187,6 +187,11 @@ def run(self):
                                                        registry_image, insecure=insecure,
                                                        force=True, dockercfg=docker_push_secret)
 
+                    if source_oci_image_path:
+                        manifests_dict = get_all_manifests(registry_image, registry, insecure,
+                                                           docker_push_secret, versions=('v2',))
+                        self.workflow.koji_source_manifest = manifests_dict['v2']
```

Comment:
`get_all_manifests` returns a dict of Responses, meaning `koji_source_manifest` here is a Response, not a dict. You are using it as a dict in `koji_util`.

Reply:
It's worse than that even: it's actually a dict of (Response, Exception) tuples.

Reply:
get_all_manifests returns dict with 'version' as key and response as value if there is response

will fix